owasp-top10-2017-apps/a6/stegonography/README.md

```diff
@@ -0,0 +1,113 @@
+# Stegonography
+
+<p align="center">
+    <img src="images/stegonography.png"/>
+</p>
+
+Stegonography is a Node.js web application that uses an API and a simple front end to simulate a real steganography application. It is built with two main routes, it's home( `/` ) and (`/login`), where the admin can log in to make changes to the page.
+
+## Index
+
+- [Definition](#what-is-security-misconfiguration)
+- [Setup](#setup)
+- [Attack narrative](#attack-narrative)
+- [Objectives](#secure-this-app-🔧)
+- [Solutions](#pr-solutions)
+- [Contributing](#contributing)
+
+## What is Security Misconfiguration?
+
+Security misconfiguration can happen at any level of an application stack, including the network services, platform, web server, application server, database, frameworks, custom code, and pre-installed virtual machines, containers, or storage. Automated scanners are useful for detecting misconfigurations, use of default accounts or configurations, unnecessary services, legacy options, etc.
+
+The main goal of this project is to discuss how **Security Misconfiguration** vulnerabilities can be exploited and to encourage developers to send secDevLabs Pull Requests  on how they would mitigate these flaws.
```

Comment:
I would rather use `app` then `project`. I mean, I see SecDevLab as the "project" and Stegonograhy as its "app".

Reply:
I agree with you! Thanks for the tip @rafaveira3!